javascript/modules/client/src/internal/client-core.ts

```diff
@@ -244,4 +257,21 @@ export abstract class ClientCore implements IClientCore {
         }
       });
   }
+
+  protected static createWithdrawAction(
+    to: string,
+    value: bigint,
+    params: IWithdraw
+  ): IProposalAction {
+    const data = ClientCore.createWithdrawActionData(params);
+    return { to, value, data };
+  }
+
+  protected static createWithdrawActionData(params: IWithdraw): string {
+    const daoInterface = DAO__factory.createInterface();
```

Comment:
Does this payload need to be generated by calling the smart contract?
For now it might be OK, but if the very same computation could be done on the client side, then I'd manually do the ABI encode packed thing that the contract does.... or create a task to do it in the future